### PR TITLE
feat: OS featurelist

### PR DESCRIPTION
**Pull request by vinckr**
Implementing the same featurelist from /pricing in the OS pages to show more detail on the capabilities of our OS projects. 
Currently only Hydra/Kratos as they are most straightforward, keto & oathkeeper are a bit harder.

- feat: kratos OS features
- feat: hydra OS features

### CODE REVIEW

**Comment by jfcurran** [comment]
right off the bat, I think this is an amazing addition - very nice 🚀 
One comment from my side would be: I think adding links to certain topics with great depth (eg OAuth2) would greatly benefit readers. these links could point to respective doc pages or in case of OAuth2 maybe even their spec.

**Review by jfcurran — APPROVED** [review]
😎

**Comment by vinckr** [comment]
Yes good point with the links!
I was debating (with myself) wether it would be good to add some or not.
Most links (except maybe OAuth2.0 spec) would lead to a ressource under ory.sh anyway... 
I will add some 😉

File: src/pages/kratos.tsx [hunk 1]
```diff
@@ -101,6 +112,61 @@ const IndexPage = () => {
...
+            ),
+            features: [
+              {
+                title: 'Self Service Login and Registration',
+                description:
+                  'Enable users to create and sign in to accounts using username/email and password combinations, Social Login, passwordless flows and more.',
+                icon: CheckCircle
+              },
+              {
+                title: 'Admin Management',
```
Comment (aeneasr) [inline src/pages/kratos.tsx:135]:
```suggestion
                title: 'User Management',
```

File: src/pages/kratos.tsx [hunk 2]
```diff
@@ -101,6 +112,61 @@ const IndexPage = () => {
...
+                description:
+                  'Manage your userbase and get, create, update or delete identities and their data.',
+                icon: CheckCircle
+              },
+              {
+                title: 'Headless APIs and Custom Data Models',
+                description:
+                  'Use customizable identity models (add fields like name, address, favourite pet ect..) and create your own user interfaces for flows in your style and branding.',
+                icon: CheckCircle
+              },
```
Comment (aeneasr) [inline src/pages/kratos.tsx:145]: Missing the UI stuff :)

File: src/pages/hydra.tsx [hunk 3]
```diff
@@ -122,6 +133,60 @@ const IndexPage = () => {
...
+            description:
+              'Issue OAuth 2.0 Access, Refresh and ID tokens in a secure and efficient manner. Ory Hydra secures critical environments for billions of users. ',
+            button: (
+              <Button style={'text'} to={'/docs'} iconRight={ArrowRight}>
+                Read the docs
+              </Button>
+            ),
+            features: [
+              {
+                title: 'Flexible User Management',
```
Comment (aeneasr) [inline src/pages/hydra.tsx:150]: Maybe more like "Integrates everywhere"?

File: src/pages/hydra.tsx [hunk 4]
```diff
@@ -122,6 +133,60 @@ const IndexPage = () => {
...
+                description:
+                  'Rely on an OpenID Certified® OIDC Provider. Ory Hydra implements all flows specified by the IETF and OpenID Foundation.',
+                icon: CheckCircle
+              },
+              {
+                title: 'Headless Interface and APIs',
+                description:
+                  'Use your own branding and user interfaces for all OAuth2.0 and OIDC flows. Ory Hydra lets you use your styles and flows powered by a REST API and intuitive CLI.',
+                icon: CheckCircle
+              },
```
Comment (aeneasr) [inline src/pages/hydra.tsx:166]: Something like "Bring your own UI" maybe?

**Comment by jfcurran** [comment]
additional note:
lets seperate content from the components and pages, and create respective props how did for `quotes.tsx` and `adopters.tsx`. 
you can create a new file in the respective folder and call it `feature-list-content.tsx` - currently (with this PR in mind) `feature-list.tsx` is being used on `pricing.tsx`, ` kratos.tsx`and `hydra.tsx`. move feature-list's respective content into `feature-list-content.tsx` and simply pass the content to feature-list using something like `featurelistPricing ` `featureslistKratos` `featurelistHydra`. that way its a bit cleaner imo

**Review by aeneasr — CHANGES_REQUESTED** [review]
Bildschirmfoto 2021-06-30 um 13 27 51

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 5]
```diff
@@ -0,0 +1,450 @@
...
+)
+
+export const featureListPricing  = {
+  features : [
+    {
+      icon: CircleThreePlus,
+      title: 'Features',
+      description: (
+        <>
+        Sign up now and instantly access these features. All features except the Ory CLI and user interfaces are included when self hosting.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:61]: This looks unformatted, can you please run `npm run format`?

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 6]
```diff
@@ -0,0 +1,450 @@
...
+}
+
+export const featureListHydra  = {
+  features : [
+    {
+      icon: CircleThreePlus,
+      title: 'Ory Hydra Features',
+      description: (
+        <>
+        Issue OAuth 2.0 Access, Refresh and ID tokens in a secure and efficient manner. Ory Hydra secures critical environments for billions of users.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:252]:
```suggestion
        Issue OAuth 2.0 Access, Refresh and ID Tokens in a secure and efficient manner. Ory Hydra secures critical environments for billions of users.
```

Comment (aeneasr) [reply src/components/layouts/featurelist/feature-list-content.tsx:252]: There are two "secure" following one another, please avoid duplication. Generally, by-sentences like "efficient manner" add little information. Maybe it makes sense to rework this sentence:

```suggestion
        Ory Hydra is a hardened and certified OAuth 2.0 and OpenID Connect provider, securing hundreds of billions of API requests in thousands of deployments.
```

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 7]
```diff
@@ -0,0 +1,450 @@
...
+            Rely on an <Button style={'link-inline'} to={'https://openid.net/developers/certified/'}> OpenID Certified® OIDC Provider</Button>. Ory Hydra implements all flows specified by the IETF and OpenID Foundation.
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'Bring Your Own UX',
+            description: (
+            <>
+            Use your own branding and user interfaces for all OAuth2.0 and OIDC flows. Ory Hydra lets you use your styles and flows powered by a REST API and intuitive CLI.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:282]:
```suggestion
            Use your own branding and user interfaces for all OAuth2.0 and OpenID Connect (OIDC) flows. Ory Hydra lets you use your styles and flows powered by a REST API and intuitive CLI.
```

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 8]
```diff
@@ -0,0 +1,450 @@
...
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'Compatible with MITreid',
+            description: (
+            <>
+            Migrate from <Button style={'link-inline'} to={'https://mitreid-connect.github.io/'}>MITREid Connect</Button> to Ory Hydra. Migration documentation is provided.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:291]: Where is the documentation?

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 9]
```diff
@@ -0,0 +1,450 @@
...
+            </>
+          ),
+            icon: CheckCircle
+          },
+          {
+            title: 'Cryptographic Key Storage',
+            description: (
+            <>
+            Store cryptographic keys for e.g. signing JWTs securely and manage OAuth 2.0 clients directly from the CLI.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:300]: We are storing these encrypted which adds secuirty

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 10]
```diff
@@ -0,0 +1,450 @@
...
+            </>
+          ),
+            icon: CheckCircle
+          },
+          {
+            title: 'Security First and High Performance',
+            description: (
+            <>
+            Sleep easy, knowing that Ory Hydra is secured against attack vectors and scales as required. Ory Hydra serves tokens to millions of users weekly and just works.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:309]: So the correct terminology here would be:

- Reduce (security) incidents
- Reduce on-call pagers

or something like that :)

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 11]
```diff
@@ -0,0 +1,450 @@
...
+
+
+export const featureListKeto  = {
+  features : [
+    {
+      icon: CircleThreePlus,
+      title: 'Ory Keto Features',
+      description: (
+        <>
+        Control who can access what in your application. Ory Keto serves decisions at scale globally with low latency.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:327]: It doesn't really serve decisions, maybe like "answers" or something like that

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 12]
```diff
@@ -0,0 +1,450 @@
...
+      button: (
+        <Button
+        style={'link'} to={'/docs/keto/quickstart/'} iconRight={ArrowRight} > Get started </Button>
+      ),
+      features: [
+        {
+          title: 'Manage and Check Permissions',
+          description: (
+            <>
+            Determine if a user is allowed to take an action with HTTP and gRPC APIs.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:339]:
```suggestion
            Determine if an entity (user, service, IoT) is allowed to perform an action using HTTP or gRPC APIs.
```

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 13]
```diff
@@ -0,0 +1,450 @@
...
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'Low Latency and Zero Downtime',
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:345]:
```suggestion
            title: 'Low Latency and Highly Available',
```

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 14]
```diff
@@ -0,0 +1,450 @@
...
+            description: (
+              <>
+              Delight users with a smooth experience and no downtime.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:348]: What is a smooth experience? Hit them with facts :)

```suggestion
              Zanzibar has maintained 95th-percentile latency of less than 10 milliseconds and availability of greater than 99.999% over 3 years of production use.
```

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 15]
```diff
@@ -0,0 +1,450 @@
...
+              </>
+            ),
+            icon: CheckCircle
+          },
+          {
+          title: 'Flexible Permission Management',
+          description: (
+            <>
+            Integrate easily with your existing data structures. Ory Keto works with any kind of identifiers.
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:357]:
```suggestion
            Ory Keto integrates with any existing data structures and identifiers.
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by jfcurran** [comment]
waiting for our friend @github-actions to give us the 👀 **preview** ✨ 
looking great locally ♥️

**Comment by vinckr** [comment]
Still need to make some changes.

File: src/components/layouts/featurelist/feature-list-content.tsx (hunk 14 above)
Comment (vinckr) [reply src/components/layouts/featurelist/feature-list-content.tsx]: How does this sound? 

Ory Keto is based on [Zanzibar](https://www.ory.sh/keto-zanzibar-evolution/), which has maintained 95th-percentile latency of less than 10 milliseconds and availability of greater than 99.999% over 3 years of production use.

Comment (aeneasr) [reply src/components/layouts/featurelist/feature-list-content.tsx]: Much better!

**Comment by aeneasr** [comment]
If a PR is work in progress, mark it as "draft" and clear the reviewers. Once it's ready to be reviewed, unmark it as draft, and tag reviewers :)

**Comment by vinckr** [comment]
@aeneasr 
ready now, only some minor changes to keto text were missing.

**Review by jfcurran — APPROVED** [review]

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 16]
```diff
@@ -0,0 +1,538 @@
...
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Zero Trust Architecture',
+          description: (
+            <>
+              Apply Zero Trust / BeyondCorp security in your tech stack. Ory
+              Oathkeeper enables secure access for all users (employees,
+              partners, customers, etc...).
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:493]: Please no `etc...` either do `item a, item b, etc.)` or `item a, item b, ...)` or `(e.g. employees, partners, customers)`

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 17]
```diff
@@ -0,0 +1,538 @@
...
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Bring Your Own UI and Identity Models',
+          description: (
+            <>
+              Use customizable identity models (add fields like name, address,
+              favourite pet ect..) and create your own user interfaces for flows
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:208]: typos

```suggestion
              favourite pet etc.) and create your own user interfaces for flows
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by aeneasr** [comment]
Please use grammarly to check your text for typos.

**Comment by aeneasr** [comment]
There are still more typos so I'm guessing you'll be tagging me again once the PR is done

File: src/components/layouts/featurelist/feature-list-content.tsx [hunk 18]
```diff
@@ -0,0 +1,538 @@
...
+              headers: X-User-ID, JSON Web Tokens and more.
+            </>
+          ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Next Generation Oathkeeper',
+          description: (
+            <>
+              <i>Coming soon:</i> The next generation of Ory Oathkeeper. Learn
```
Comment (aeneasr) [inline src/components/layouts/featurelist/feature-list-content.tsx:522]: No `<i>` tags, please use `<em>` instead:

```suggestion
              <em>Coming soon:</em> The next generation of Ory Oathkeeper. Learn
```

**Pull request merged**